app/models/service_template_ansible_playbook.rb

```diff
@@ -1,6 +1,7 @@
 class ServiceTemplateAnsiblePlaybook < ServiceTemplateGeneric
   before_destroy :check_retirement_potential, :prepend => true
   around_destroy :around_destroy_callback, :prepend => true
+  after_rollback :rollback_job_templates, :on => :create
```

Comment:
which create - new `ServiceTemplate`? Even on update we can end up creating new job template which may need to be rolled back too.